lib/uksched/Config.uk

```diff
@@ -7,12 +7,17 @@ menuconfig LIBUKSCHED
 	select HAVE_SCHED
 
 if LIBUKSCHED
-	# Invisible symbol to enable TCB initialization
-	config LIBUKSCHED_TCB_INIT
-		bool
-		default n
 
-	config LIBUKSCHED_DEBUG
-		bool "Enable debug messages"
-		default n
+# Invisible symbol to enable TCB initialization
+config LIBUKSCHED_TCB_INIT
+	bool
+
+config LIBUKSCHED_DEBUG
+	bool "Enable debug messages"
+
+config LIBUKSCHED_STATS
+	bool
+	select LIBUKLIBID
```

Comment:
Why is `UKLIBID` selected here? Won't it be active when [`UKSTORE`](https://github.com/unikraft/unikraft/blob/staging/lib/ukstore/Config.uk#L5) is selected?

Reply:
Generally speaking, dependencies must be defined explicitly. This applies to both Kconfig and headers. Saying that, in this particular case I don't see why `uklibid` is there, most probably it's something left as things changed during development. Thanks for spotting it out 👍🏼